### PR TITLE
Use ERDF logo from asset-manager

### PR DESCRIPTION
**Pull request by tijmenb**
In #2280 we made the detailed guides use the logo URL from the database. Using assets from `app/assets` directly will cause problems because there is no reference in the code to this image anymore, so the image could be deleted in the future.

This commit changes the database URL to a static, never-to-be-deleted, version uploaded via the asset-manager.

cc @edds

### CODE REVIEW

File: db/data_migration/20150716152627_add_static_eu_logo_to_regional_development_funding.rb [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+  'erdf-national-guidance',
+  'previous-erdf-programmes-retaining-documents',
+  'european-territorial-cooperation-programmes',
+  'reuniting-europe-programme-turkey'
+]
+logo_url = "https://assets.digital.cabinet-office.gov.uk/media/55a7cc72ed915d5374000001/erdf-logo.png"
+
+puts "Adding logos to ERDF editions"
+slugs.each do |slug|
+  document = Document.find_by(slug: slug)
```
Comment (alext) [inline db/data_migration/20150716152627_add_static_eu_logo_to_regional_development_funding.rb]: You should be passing a `document_type` here as well.  slugs are only unique per document_type.

Comment (tijmenb) [inline db/data_migration/20150716152627_add_static_eu_logo_to_regional_development_funding.rb]: @alext  Updated PR.

Also verified on preview that the slugs are only for Detailed Guides, so nothing went wrong there.

```
irb(main):029:0> Document.where(slug: [
irb(main):030:2*   'erdf-programmes-and-resources',
irb(main):031:2*   'erdf-programmes-progress-and-achievements',
irb(main):032:2*   'applying-for-erdf-funding',
irb(main):033:2*   'erdf-national-guidance',
irb(main):034:2*   'previous-erdf-programmes-retaining-documents',
irb(main):035:2*   'european-territorial-cooperation-programmes',
irb(main):036:2*   'reuniting-europe-programme-turkey'
irb(main):037:2> ]).map(&:document_type).uniq
  Document Load (0.4ms)  SELECT `documents`.* FROM `documents` WHERE `documents`.`slug` IN ('erdf-programmes-and-resources', 'erdf-programmes-progress-and-achievements', 'applying-for-erdf-funding', 'erdf-national-guidance', 'previous-erdf-programmes-retaining-documents', 'european-territorial-cooperation-programmes', 'reuniting-europe-programme-turkey')
=> ["DetailedGuide"]
```

**Comment by alext** [comment]
:+1: Looks good to me (assuming the tests are green)

**Comment by edds** [comment]
Thanks for doing this @tijmenb. Also :+1: for deleting the asset at the same time.

**Pull request merged**